### DISCUSSION TITLE
2.1.1rc1

### DISCUSSION THREAD

**Discussion by eschutho** [apache/superset#24435] [General]
Please comment regarding any questions or bugs that you may have found for this release, and remember to vote in the email thread!

-----------
The release candidate:
https://dist.apache.org/repos/dist/dev/superset/2.1.1rc1/

Git tag for the release:
https://github.com/apache/superset/tree/2.1.1rc1

The Change Log for the release:
https://github.com/apache/superset/blob/2.1.1rc1/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/2.1.1rc1/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/2.1.1rc1'>2.1.1rc1</a>.</em>

**Comment by sfirke** [comment]
_Also sent to the email thread, posting here for greater visibility._

I tested 2.1.1rc1.  The addition of [strong session protection by default](https://github.com/apache/superset/pull/24256) meant that I could not log into Superset when I switched the image to 2.1.1rc1 – I was stuck in a redirect loop from the welcome screen to the login screen.  I changed my config to SESSION_PROTECTION = “basic” and everything worked okay after that.  Iincluding alerts & reports, which some users report is broken by the default value of “strong”, see the prior link.

Is that enough of a breaking change that the default setting of “strong” should happen in 3.0.0?  If it’s going into 2.1.1, I think it should be highlighted prominently on [UPDATING.md](https://github.com/apache/superset/blob/master/UPDATING.md).  Right now that page does not have a section for 2.1.1, just “Next” which is mostly 3.0.0 changes.

Otherwise the release looks good.  The bug fixes that pertain to my issues work great (like the PivotTable v2 width correction)!

**Reply by juanmanuelramallo** [reply]
We just upgraded from 2.1.0 to 2.1.1rc1 with the new `SESSION_PROTECTION = "strong"` configuration. After deploying, I was logged out (expected) but I was able to log in at first time without any issues.

**Reply by sfirke** [reply]
Hm, I just double-checked -- restarted Superset, used a different browser, I get the same stuck-in-a-loop behavior.  I am trying to think what could be different -- cookie settings?  Auth method?  I'm using OAuth.

**Reply by juanmanuelramallo** [reply]
I'm not using OAuth, just username and password. 

And here are my cookie settings:
```python
SESSION_COOKIE_HTTPONLY = True  # Prevent cookie from being read by frontend JS?
SESSION_COOKIE_SECURE = True  # Prevent cookie from being transmitted over non-tls?
SESSION_COOKIE_SAMESITE: Optional[Literal["None", "Lax", "Strict"]] = "None"
SESSION_PROTECTION = "strong"
```

**Comment by bui-theanh** [comment]
hello, What image version should I use for your version, please tag the image you use ^^

**Comment by mdeshmu** [comment]
Please include https://github.com/apache/superset/pull/24709

**Comment by OskarSorensen** [comment]
Please consider including PR #18924 for this RC

**Comment by LeoDiep** [comment]
can we fix this as well in version 2.1.1
https://github.com/apache/superset/issues/23735
